src/main/fc/rc_adjustments.h

```diff
@@ -131,6 +135,7 @@ typedef enum {
     ADJUSTMENT_TYPE_PROF  = BIT(1),
     ADJUSTMENT_TYPE_GOV   = BIT(2),
     ADJUSTMENT_TYPE_MIX   = BIT(3),
+    ADJUSTMENT_TYPE_ACC   = BIT(4),
```

Comment:
TYPE_NONE is for the parameters that don't need initialisation.
Don't add a new one for the purpose.

Reply:
Ok, good to know. It has been fixed.